### DISCUSSION TITLE
使用了一段时间了，近日不行了

### DISCUSSION THREAD

**Discussion by codez-yydd** [simple-robot/Discussions#104] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

code=238, title=禁止登录, message=请使用手Q扫描二维码或短信登录
最早是ANDROID_PHONE使用了2-3天，然后使用ANDROID_WATCH稳定的很不错，从前天开始出现这个错误
无论我使用以下哪个都无法登陆，手表是238，其他则是版本低，需要更新
ANDROID_PHONE
ANDROID_PAD
ANDROID_WATCH
IPAD
MACOS

**Comment by ForteScarlet** [comment]
参考
- https://github.com/simple-robot/simpler-robot/issues/617
- https://github.com/simple-robot/simpler-robot/issues/614

其他参考：mirai相关社区中的一切相关内容。

_风控和版本低都是老大难问题_

**Reply by zuijiaosy** [reply]
mirai 已经有插件能解决 但是simple bot不支持插件这就很恼火了

**Reply by ForteScarlet** [reply]
如果真想用插件可以去找找那种可以在 mirai-core 中加载插件的第三方库，可以用来使用mirai插件

印象中曾经在GitHub见过这种仓库

**Comment by enchantedyou** [comment]
我也遇到了
